docs/snippets/GPU_Metric1.cpp

```diff
@@ -0,0 +1,15 @@
+#include <ie_core.hpp>
+
+int main() {
+using namespace InferenceEngine;
+//! [part0]
+InferenceEngine::Core core;
+CNNNetwork cnnNetwork = core.ReadNetwork(FLAGS_m);
+std::map<std::string, Parameter> options = {{"CNN_NETWORK", &cnnNetwork}}; // Required. Set the address of the target network.
+options.insert(std::make_pair("GPU_THROGHPUT_STREAMS", 2)); // Optional. Set only when you want to estimate max batch size for a specific throughtput streams. Default is 1 or throughtput streams set by SetConfig.
+options.insert(std::make_pair("BASE_BATCH_SIZE", 32)); // Optional. Set only when you want to estimate max batch size using a specific batch size. (default is 16).
```

Comment:
Do we really need it in user API? We shouldn't make users brute force the values for this parameter

Reply:
Then how about not mentioning this in the doc but having internally for experimental purpose?

Reply:
Maybe it should be debug option then?

Reply:
Changed this to be applied with debug config